checks/raw/binary_artifact_test.go

```diff
@@ -21,54 +21,136 @@ import (
 
 	"github.com/golang/mock/gomock"
 
+	"github.com/ossf/scorecard/v4/clients"
 	mockrepo "github.com/ossf/scorecard/v4/clients/mockclients"
 )
 
+func strptr(s string) *string {
+	return &s
+}
+
 // TestBinaryArtifact tests the BinaryArtifact checker.
 func TestBinaryArtifacts(t *testing.T) {
 	t.Parallel()
 	tests := []struct {
-		name   string
-		err    error
-		files  []string
-		expect int
+		name                   string
+		err                    error
+		files                  [][]string
```

Comment:
could this be 
```
   files       []string
   actionFiles []string
```
instead?

Reply:
I see what you mean. Eventually, once #2048 is solved and the new test cases are moved to checks/evaluation from checks/raw, it'll no longer be helpful to use [][]string here.

In the meantime, with the current test cases, it might make sense to use [][]string because each []string represents a ListFiles() call